docs/.vuepress/config.js

```diff
@@ -64,11 +64,11 @@ module.exports = {
         },
         {
           service: "linkedin",
-          url: "https://www.linkedin.com/company/ignt/",
+          url: "https://www.linkedin.com/company/ignite/",
```

Comment:
```suggestion
          url: "https://www.linkedin.com/company/ignt/",
```

Reply:
not anymore! Per Candice we are going to be Ignite on LinkedIn